scripts/prover_profile.sh

```diff
@@ -0,0 +1,48 @@
+
+# Simple script to run the generic prover with some keys and witness data,
+# recording the output (including timing information) to separate files.
+#
+# File names below assume that the key and witness file are named
+#   ${CIRCUIT}_pk.bin
+#   ${CIRCUIT}_vk.bin
+#   ${CIRCUIT}_assignment.bin
+#   ${CIRCUIT}_primary.bin
+# in the directory ${DATA_DIR} (set on command line or env).
+#
+# The zeth-tool binary is expected to be in the path, or at ${ZETH_TOOL}.
+
+# Set default DATA_DIR
+[ "${DATA_DIR}" == "" ] && DATA_DIR=~/dev/neodizk/test_data/zeth
```

Comment:
As per my previous comment https://github.com/clearmatics/zeth/pull/385#pullrequestreview-731005028, if this script is to be mostly used in the context of neodizk<>zeth (which is suggested by the default value of `DATA_DIR`), I'd add more color here on the value of DATA_DIR. This can either be done by adding a line of comment above, or by moving this script in a `scripts/neodizk` folder and providing the necessary context in a README. If we want this script to be rather generic and used in several contexts, then I'd refrain from pointing to a location under `neodizk` (which is rather confusing) and would rather use a default path that is under `zeth/` and maybe add a comment to explain why we may want to point to a `neodizk` location in some circumstances.

Reply:
> As per my previous comment #385 (review), if this script is to be mostly used in the context of neodizk<>zeth 

This script profiles the code in this repository, so no need to reference external projects.   I've removed the default values.

Reply:
Thanks, I think this is cleaner that way, else you are left wondering why the default value points to a `neodizk` location. If we want to use a default value that's fine but it should rather point to the `testdata` folder of zeth, though that's fine like this, users will set the env var manually to fit their config.